library/common/config/config.cc

```diff
@@ -36,6 +36,7 @@ const std::string config_header = R"(
 - &dns_refresh_rate 60s
 - &dns_fail_base_interval 2s
```

Comment:
@goaway in slack we discussed 25s timeout + 5s failure refresh rate. The rate as is here starts a 2s and backoffs to 10s. Are we ok with that policy or do you want to reduce max to 5s?

Reply:
I think that policy is fine for now.